From 9e424232b383461f4a1bc4d73fbe15cb202b99b0 Mon Sep 17 00:00:00 2001
From: Lincoln Wallace <lincoln.wallace@canonical.com>
Date: Wed, 11 Sep 2024 19:52:20 -0300
Subject: [PATCH 1/1] test: running for devmode stress-ng snap

Signed-off-by: Lincoln Wallace <lincoln.wallace@canonical.com>
---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 2862bfa39..59cb13162 100644
--- a/Makefile
+++ b/Makefile
@@ -877,12 +877,12 @@ lite-test: all
 	STRESS_NG=./stress-ng debian/tests/lite-test
 
 .PHONY: slow-test-all
-slow-test-all: all
-	./stress-ng --seq 0 -t 15 --pathological --times --tz --metrics --klog-check --progress --cache-enable-all -x smi || true
+slow-test-all:
+	/snap/bin/stress-ng --seq 0 -t 15 --pathological --times --tz --metrics --klog-check --progress --cache-enable-all -x smi || true
 
 .PHONY: verify-test-all
-verify-test-all: all
-	./stress-ng --seq 0 -t 5 --pathological --times --tz --metrics --verify --progress --cache-enable-all -x smi || true
+verify-test-all:
+	/snap/bin/stress-ng --seq 0 -t 5 --pathological --times --tz --metrics --verify --progress --cache-enable-all -x smi || true
 
 .PHONY: tags
 tags:
-- 
2.43.0

